packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionStart.test.ts

```diff
@@ -430,46 +327,6 @@ describe('Force Function Start', () => {
         nls.localize('force_function_start_no_org_auth')
       );
     });
-
-    it('Should handle cancellation requested', async () => {
```

Comment:
I couldn't figure out how to mock and trigger cancellation for the progress bar provided by `LibraryCommandletExecutor`